### DISCUSSION TITLE
How to implement the decimal function in DB2 within Apache Doris?

### DISCUSSION THREAD

**Discussion by ITxin97** [apache/doris#38955] [A - General / Q&A]
To call the CAST function for this purpose, for example, to achieve DECIMAL(2.2, 2, 1) by calling CAST(2.2 AS DECIMAL(2, 1)), how can this be implemented at the code level?
DB2 Documentation[https://www.ibm.com/docs/en/db2/11.5?topic=functions-decimal-dec](url)

**Comment by maruyue** [comment]
https://doris.apache.org/docs/sql-manual/sql-functions/cast

**Comment by maruyue** [comment]
CAST(col as DECIMAL (x, y)）is supported
